soccer/src/soccer/strategy/agent/position/waller.cpp

```diff
@@ -0,0 +1,42 @@
+#include "waller.hpp"
+
+namespace strategy {
+
+Waller::Waller() { std::string defense_type_{"Waller"}; }
```

Comment:
Here, defense_type_ should be a [private member variable](https://www.learncpp.com/cpp-tutorial/public-vs-private-access-specifiers/). See `position.hpp` for an example. The case issue will fix itself once you make this private.

Also, please add a constructor for this class: https://www.learncpp.com/cpp-tutorial/constructors/